BoxContentSDK/BoxContentSDK/Operations/BOXAPIDataOperation.h

```diff
@@ -19,10 +19,10 @@ typedef void (^BOXAPIDataProgressBlock)(long long expectedTotalBytes, unsigned l
 /**
  * BOXAPIDataOperation is a concrete subclass of BOXAPIAuthenticatedOperation.
  * This operation receives binary data from the Box API which may be in the form
- * of downloads or thumbnails.
+ * of downloads or thumbnails. Thumbnail downloads are performed via BOXAPIThumbnailDownloadDataOperation which are runned on a specific queue.
```

Comment:
* of downloads.
* Thumbnail downloads are performed via a BOXAPIThumbnailDataOperation which run on a specific queue.